server/index.js

```diff
@@ -21,8 +20,10 @@ app.get('/products/dept/:dept', (req, res) => {
 });
 
 app.get('/products/brand/:brandName', (req, res) => {
-  let firstBrandWord = formatName(req.params.brandName).split(' ')[0];
-  RecommendedItem.find({ brand: {$regex: '^' + firstBrandWord}}, (err, results) => {
+  // escape certain characters from request url
+  let brandWords = req.params.brandName.split(/[,.\s-&amp]/);
+  RecommendedItem.find({
+    brand: {$regex: `^${brandWords.join('.*\s*')}$`, $options: 'i'}}, (err, results) => {
     res.json(results);
   });
```

Comment:
This may be because I'm unfamiliar with Regex, but what are you doing here? It seems like you are escaping an s an doing something with  $options: 'i'.

Reply:
yeah, there are a couple of pieces to it:

- `.*` allows any number of any type of character
- `\s*` allows any number of whitespace characters
- `$options: 'i'` makes it case insensitive

This makes it so a formatted company name (e.g. Hudson, Lakin, and Clay) would be found in the db even if `req.params.brandName` was something like `hudson-lakin-and-clay`

Reply:
ooh I see, that's pretty useful. Thanks!